docs/backend-api/resources/tracking/tracker/engine_immobilizer.md

```diff
@@ -1,37 +1,84 @@
 ---
-title: /engine_immobilizer
-description: /engine_immobilizer
+title: Engine immobilizer
+description: Engine immobilizer
 ---
+# Engine immobilizer
 
 API base path: `/tracker/engine_immobilizer`
 
+Engine immobilizer is an electronic security device fitted to a motor vehicle that prevents the engine from running unless it must run. This prevents the vehicle from being "hot wired" after entry has been achieved and thus reduces motor vehicle theft. This API call allows manipulating with immobilizer state.
+
 ### read
+
 Request to read the state of engine immobilizer.
 
 #### parameters
-* **tracker_id** - **int**. Id of the tracker (aka “object_id”). Tracker must belong to authorized user and not be blocked.
+
+| name | description | type| format|
+| :------ | :------ | :----- | :------ |
+| tracker_id | Id of the tracker (aka “object_id”). Tracker must belong to authorized user and not be blocked. | int | 123456 |
+
+#### examples
+
+=== "HTTP POST application/json
```

Comment:
Missing closing quote. I suggest using `cURL` and `HTTP GET` for tab titles to make it more clear.

Reply:
just to confirm 
1) we use `shell` for curl and `adap` for GET
2) we use the next names:
"HTTP POST application/json" -> "cURL"
and
"GET" -> "HTTP GET"?